edxval/models.py

```diff
@@ -268,7 +268,7 @@ class VideoImage(TimeStampedModel):
     generated_images = ListField()
 
     @classmethod
-    def create_or_update(cls, course_video, file_name, image_data=None):
+    def create_or_update(cls, course_video, file_name, image_data=None, generated_images=None):
```

Comment:
can we also make `file_name` a keyword argument? as in case of `generated_images` it's not needed.